projects/crosscurve/index.js

```diff
@@ -0,0 +1,106 @@
+const { sumTokensExport } = require("../helper/unwrapLPs");
+
+module.exports = {
+  ethereum: {
+    tvl: sumTokensExport({
+      owner: "0xac8f44ceca92b2a4b30360e5bd3043850a0ffcbe",
+      tokens: [
```

Comment:
hi @spiehdid thanks for the PR, is there some way to pull the token list on chain? like from this whitelist contract? https://etherscan.io/address/0x5cF21A9899e679448284427C0Baca807fAd6bB8F#code

I tried to use `tokens` but for some reason, I keep getting error

Reply:
There are a lot of extra tokens

Reply:
sorry, I dont understand. you mean, there are more tokens than those that are part of the whitelist?

Reply:
There are many tokens in the white list that should not be included in the TVL